pkg/ipmi/get_sensor_reading.go

```diff
@@ -12,6 +12,9 @@ import (
 type GetSensorReadingReq struct {
 	layers.BaseLayer
 
+	// Use the value from FullSensorRecord.OwnerLUN.
```

Comment:
The request was intended to be that layer in the UDP payload. What do you think about putting this field in the `GetSensorReadingCmd` struct?

Reply:
Sounds good to me. I had to rename the field so as to avoid overlapping with the function's name.